### PR TITLE
Send sms from sim module

### PR DESCRIPTION
**Pull request by KatrideRRR**

### CODE REVIEW

File: sim_module_test_sketch/sim_module_test_sketch.ino [hunk 1]
```diff
@@ -1,38 +1,41 @@
 #include <SoftwareSerial.h>
+SoftwareSerial mySerial(3, 2); //SIM800L Tx & Rx is connected to Arduino D3 & D2
 
-//Create software serial object to communicate with SIM800L
-SoftwareSerial mySerial(3, 2);  //SIM800L Tx & Rx is connected to Arduino D3 & D2
-
-void setup() {
-  //Begin serial communication with Arduino and Arduino IDE (Serial Monitor)
```
Comment (arsen-adzhiametov) [inline sim_module_test_sketch/sim_module_test_sketch.ino:7]: Please revert unnecessary changes like this

File: sim_module_test_sketch/sim_module_test_sketch.ino [hunk 2]
```diff
@@ -1,38 +1,41 @@
...
+void setup()
+{
   Serial.begin(9600);
-
-  //Begin serial communication with Arduino and SIM800L
+  
   mySerial.begin(9600);
 
-  Serial.println("Initializing...");
+  Serial.println("Initializing...");
```
Comment (arsen-adzhiametov) [inline sim_module_test_sketch/sim_module_test_sketch.ino]: Here you just added redundant space. We do not need to have such unnecessary changes

**Review by arsen-adzhiametov — CHANGES_REQUESTED** [review]
Please revert unnecessary changes

File: sim_module_test_sketch/sim_module_test_sketch.ino (hunk 1 above)
Comment (KatrideRRR) [reply sim_module_test_sketch/sim_module_test_sketch.ino:7]: Fixed.

File: sim_module_test_sketch/sim_module_test_sketch.ino (hunk 2 above)
Comment (KatrideRRR) [reply sim_module_test_sketch/sim_module_test_sketch.ino]: Fixed.

**Review by arsen-adzhiametov — APPROVED** [review]
